frontend/src/v5/ui/components/shared/linkCard/linkCard.styles.ts

```diff
@@ -19,20 +19,16 @@ import { Typography } from '@controls/typography';
 import { Link as DecoratedLink } from 'react-router-dom';
 import { Card, CardContent } from '@mui/material';
 import styled, { css } from 'styled-components';
-import { Avatar } from '@controls/avatar';
 
 export const ListItem = styled.li`
```

Comment:
can we rename this to `CardListItem` ?

Reply:
Also Link to CardLink

Reply:
I will proceed to rename ListItem to CardListItem, but I wonder id the second one may be confusing since the component itself is already called `LinkCard` and it would contain a `CardLink` component

Reply:
Yeah, I think taking out the Card prefix might be better. These are subcomponents only used for LinkCard, right?

Reply:
Mostly. The `ListItem` is used also by the `AddProjectCard` and `AddTeamspaceCard`, and the `StyledCard` is also used by the `TeamspacePlaceholderCard`, but generally speaking, they all are LinkCards in a sense